test/integration_test_helper.rb

```diff
@@ -3,7 +3,7 @@
 require "capybara-select-2"
 require "support/govuk_test"
 
-class ActionDispatch::IntegrationTest
+class PublisherIntegrationTest < ActionDispatch::IntegrationTest
```

Comment:
I see why we cant just name the class as 'IntegrationTest' but I am not sure about 'PublisherIntegrationTest'. its more like a helper to me but inheriting from helper would also not make much sense. may be a bit more generic like 'AppIntegrationTest' ?

Reply:
We probably could rename this to be `IntegrationTest`, if desired (and the legacy one to `LegacyIntegrationTest`?

I don't really understand how `AppIntegrationTest` is any different to `PublisherIntegrationTest` (in meaning)?